components/list/image/src/index.js

```diff
@@ -0,0 +1,97 @@
+import React, {PropTypes} from 'react'
+import SuiLazyLoadImage from '@schibstedspain/sui-image-lazy-load'
+import cx from 'classnames'
+
+const ensureFunction = (func) => {
+  return func instanceof Function ? func : () => {
+  }
+}
+
+/**
+ * Render a simple sui-image-lazy-load component.
+ * @param image {Object} src, alt and onClick callback configuration of the image to display.
+ */
+const renderSuiLazyloadImage = (image) => (
+  <SuiLazyLoadImage
+    debounce={false}
+    offsetVertical={100}
+    src={image.src}
+    alt={image.alt}
+    showSpinner
+    aspectRatio='1:1' />
+)
+
+/**
+ * Creates a simple list item containing a regular image with the configuration provided.
+ * @param image {Object} src, alt and onClick callback configuration of the image to display.
+ * @param index {Number} Order index of the li element to create.
+ */
+const renderImageItem = (image, index) => (
+  image && <li className='sui-ListImage-item' key={index} onClick={ensureFunction(image.onClick)}>
+    {renderSuiLazyloadImage(image)}
+  </li>
+)
+
+/**
+ * Creates an special list item, which has a regular image and an additional cover box over it (lack 65% opacity
```

Comment:
Consider removing that 65% as it's not really set in this component

Reply:
Done! 👍